htdocs/core/boxes/box_factures_imp.php

```diff
@@ -77,8 +77,10 @@ function loadBox($max=5)
             $sql.= " f.tva as total_tva,";
             $sql.= " f.total_ttc,";
 			$sql.= " f.paye, f.fk_statut, f.rowid as facid";
+			$sql.= ", sum(pf.amount) as am";
```

Comment:
If you introduce a sum into a sql request, you must also add a group by for all fields except sum, to be Sql standard compliant